addons/pharma/stringtable.xml

```diff
@@ -3917,6 +3996,7 @@
             <Dutch>Vereist medisch niveau voor gebruik van IV Adrenaline</Dutch>
             <French>Niveau médical requis pour l'épinéphrine IV</French>
             <Italian>Livello medico richiesto per Epinefrina IV</Italian>
+            <German>Medizinisches Niveau für intravenös verabreichtes Epinephrine erforderlich</German>
```

Comment:
Same as with the Coag-Sense.
"Erforderliches medizinisches Niveau für die Gabe von IV-Epinephrine"